internal/oci/runtime_vm.go

```diff
@@ -318,6 +325,14 @@ func (r *runtimeVM) ExecSyncContainer(ctx context.Context, c *Container, command
 		return nil, errors.Wrap(err, "ExecSyncContainer failed")
 	}
 
+	// if the execution stopped because of the timeout, report it as such
+	if exitCode == execTimeout {
+		return &types.ExecSyncResponse{
+			Stderr:   []byte(conmonconfig.TimedOutMessage),
+			ExitCode: -1,
```

Comment:
Is this one not changed on purpose?

Reply:
Yes. According to the [equivalent code un runtime_oci.go](https://github.com/cri-o/cri-o/blob/master/internal/oci/runtime_oci.go#L369), this is what the caller expects.